### PR TITLE
[SMALL] [Android QS| 00-Login] The Logout and Login values inside the String xml are missing.

### PR DESCRIPTION
**Pull request by benlizar**
https://auth0team.atlassian.net/browse/QS-385

### CODE REVIEW

File: 00-Login/app/src/main/res/values/strings.xml.example [hunk 1]
```diff
@@ -3,4 +3,6 @@
 
     <string name="com_auth0_client_id">{CLIENT_ID}</string>
     <string name="com_auth0_domain">{DOMAIN}</string>
+    <string name="login">Login</string>
+    <string name="logout">Logout</string>
```
Comment (lbalmaceda) [inline 00-Login/app/src/main/res/values/strings.xml.example]:
```suggestion
    <string name="logout">Log out</string>
```

File: 00-Login/app/src/main/res/values/strings.xml.example [hunk 2]
```diff
@@ -3,4 +3,6 @@
 
     <string name="com_auth0_client_id">{CLIENT_ID}</string>
     <string name="com_auth0_domain">{DOMAIN}</string>
+    <string name="login">Login</string>
```
Comment (lbalmaceda) [inline 00-Login/app/src/main/res/values/strings.xml.example]:
```suggestion
    <string name="login">Log in</string>
```

File: 03-Session-Handling/app/src/main/res/values/strings.xml.example [hunk 3]
```diff
@@ -1,7 +1,7 @@
 <resources>
     <string name="app_name">Session Management</string>
     <string name="refresh">Refresh the access_token</string>
-    <string name="login">Log in</string>
+    <string name="login">Login</string>
     <string name="logout">Logout</string>
```
Comment (lbalmaceda) [inline 03-Session-Handling/app/src/main/res/values/strings.xml.example]:
```suggestion
    <string name="logout">Log out</string>
```

**Comment by benlizar** [comment]
Fixed all commentaries. @lbalmaceda I will review all other QS to see if there is another QS using a noun instead of Verb.

**Pull request merged**